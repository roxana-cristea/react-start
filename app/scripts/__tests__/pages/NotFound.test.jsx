import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from '../../pages/NotFound';

describe('<NotFound />', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<NotFound />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
