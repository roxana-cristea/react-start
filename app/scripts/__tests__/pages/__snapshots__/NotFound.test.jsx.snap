// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<div
  className="not-found-container"
>
  <div
    className="title"
  >
    Page not found ;)
  </div>
</div>
`;
